staking/programs/staking/src/lib.rs

```diff
@@ -534,4 +534,38 @@ pub mod staking {
             Err(error!(ErrorCode::DebuggingOnly))
         }
     }
+
+    pub fn request_split(ctx: Context<RequestSplit>, amount: u64, recipient: Pubkey) -> Result<()> {
+        ctx.accounts.stake_account_split_request.amount = amount;
+        ctx.accounts.stake_account_split_request.recipient = recipient;
+        Ok(())
+    }
+
+
+    pub fn accept_split(ctx: Context<AcceptSplit>) -> Result<()> {
+        // TODO : Split vesting schedule between both accounts
+
+        // TODO : Transfer stake positions to the new account if need
```

Comment:
I think you probably want to add the logic for ^ to this PR before merging, or make this instruction a noop so that the committed version of the code can't produce accounts in an inconsistent state. (i.e., there would be no risk to deploying that version of the code)

Reply:
Throwing error for now